modules/gitops-argo-cd-properties.adoc

```diff
@@ -108,6 +146,8 @@ a|* `<autotls>` - Use the provider to create the Redis server's TLS certificate.
 
 |`resourceInclusions` |The configuration to identify which resource group/kinds are applied.|`__<empty>__` |
 
+|`ResourceTrackingMethod` |The field used by Argo CD to monitor of its managed resources.|`__<label>__` |
```

Comment:
Should begin with lowercase letter, here, and all other fields in this document that begin with uppercase letters.

Reply:
Fixed.